### ISSUE TITLE
Update edit profile with interests

### ISSUE THREAD

**Issue by asimonw** [emfoundation/ce100-app#1315]
"As a CE100 contact, I want to be able to edit my interest tags at any time, so that what people can find out about me as well as my feed of matches are always up to date and relevant."

The profile edit form will contain an additional section to add or edit interest tags. The mechanism and UI will be exactly the same as for org and challenge tags.

**Comment by fannyBkane** [comment]
Hey @Danwhy is there a reason why the following wasn't taken into account?

> Note: the mockup shows layout and other styling changes. Only take into account the grouping of fields (with corresponding headings) in this issue.

**Comment by sophielevens** [comment]
Hi @fannyBkane, we thought you meant to ignore the styling changes and just take into account the addition of the interest field section. Was the restyling of the edit profile form part of this issue too?

**Comment by fannyBkane** [comment]
Hey @sophielevens that's right, please ignore the styling changes. But we wanted to group fields and add headers, as shown on the mockup (edit profile, manage notifications and manage account). Apologies if that wasn't clear in the first place :/

**Comment by sophielevens** [comment]
Hey @fannyBkane, for admin users would you like user type and organisation to be within the 'manage account' section? 

![image](https://user-images.githubusercontent.com/25961258/41229151-e44e56fc-6d72-11e8-8c78-2eaad602ca0a.png)

Thanks 👍

**Comment by fannyBkane** [comment]
Hey @sophielevens yes absolutely, that makes sense. As I am not sure it was clearly part of the scope, it would be great if @asimonw could confirm it's ok to update the admin form?

**Comment by asimonw** [comment]
@sophielevens @fannyBkane Yes that makes sense. Not strictly part of the initial scope, but good to keep things consistent!

**Comment by fannyBkane** [comment]
Hey @sophielevens for admins there doesn't seem to be a 'Manage account' header, and the reset password link has also disappeared
 
![screen shot 2018-06-14 at 17 02 39](https://user-images.githubusercontent.com/20577339/41424046-dbf1d13a-6ff4-11e8-9172-7737f08b94de.png)
